### PR TITLE
Add Codeship Jet v1.4.0

### PR DESCRIPTION
**Pull request by mlocher**
New cask for the Codeship Docker CLI, see https://codeship.com/documentation/docker/ for more information.
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by adidalal** [comment]
@vitorgalvao Closed-source, but CLI-only. Think it goes here or `homebrew-binary`?

**Comment by vitorgalvao** [comment]
@adidalal I’d say here, `homebrew-binary` seems mostly stale, and whatever they have that would be better served by us should likely move.

Have you found the download URL to verify?

**Comment by adidalal** [comment]
Yes, verified from https://codeship.com/documentation/docker/installation/

File: Casks/jet.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'jet' do
+  version '1.4.0'
+  sha256 '49462f143e3948a49c984a91a472089ecb9c14bc2d29415e5c4278e9d9dd4a63'
+
+  url "https://s3.amazonaws.com/codeship-jet-releases/#{version}/jet-darwin_amd64_#{version}.tar.gz"
```
Comment (vitorgalvao) [inline Casks/jet.rb]: Needs a `# s3.amazonaws.com/codeship-jet-releases was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by adidalal** [comment]
Should probably have a dependency on docker-toolbox or at least docker

**Comment by mlocher** [comment]
Thanks for the review, I added the comment and a `depends_on` for Docker. I'd be happy to rebase on the current `master` branch and squash the two commits if you're fine with the formula as it is right now :smile:

**Comment by adidalal** [comment]
Yes, please do. Thanks!

**Pull request merged**